src/styles/header.scss

```diff
@@ -0,0 +1,113 @@
+header {
+    position: fixed;
+    z-index: 999;
+    width: 100%;
+    height: 3rem;
+    min-height: 50px;
+    color: #fff;
+    background: rgba(0, 8, 81, 0.8);
+    border: 1px solid transparent;
+    border-radius: 0;
+
+    .input-group {
+      float: right;
+      width: 200px;
```

Comment:
-  Uniformity among the CSS relative units `{px, rem, and em, %}`. Because of the DPI of displays, I will think about using `{rem}` or `{em}`.